build/assets/themes/dashboard/js/file-manager/file-manager.js

```diff
@@ -162,8 +162,9 @@ import Dropzone from '../../../../../node_modules/dropzone/dist/dropzone';
         my.$searchResultMenu.find('a[data-file-manager-action=download]').on('click', function(e) {
             var fID = $(this).data('file-id');
             e.preventDefault()
+            var fUUID = $("input[data-item-id=" +fID + "]").data("item-uuid")
             window.frames['ccm-file-manager-download-target'].location =
-                CCM_DISPATCHER_FILENAME + '/ccm/system/file/download?fID=' + fID
+                CCM_DISPATCHER_FILENAME + '/ccm/system/file/download?fID=' + fUUID
```

Comment:
Won't this break if the user hasn't upgraded their site files UUID? It looks like the backend file download() method will accommodate either value, but I only see fUUID being passed here.

Reply:
is fixed now.